cluster/images/hyperkube/Makefile

```diff
@@ -16,8 +16,8 @@
 #
 # Usage:
 #   [ARCH=amd64] [REGISTRY="gcr.io/google_containers"] make (build|push) VERSION={some_version_number e.g. v1.2.0}
-
-REGISTRY?=gcr.io/google_containers
+VERSION=v1.3.0
+REGISTRY?=taimir93
```

Comment:
I used my private registry for testing, will change it back when we remove the [WIP] tag.